.github/workflows/main.yml

```diff
@@ -1,40 +0,0 @@
-name: Syncing Your Cabin
```

Comment:
You'll have to commit this back in because the bot removed it (remember to uninstall the bot when you're working on a course)